solidity/contracts/bridge/Bridge.sol

```diff
@@ -74,15 +92,37 @@ contract Bridge {
     struct DepositInfo {
         // Ethereum depositor address.
         address depositor;
-        // Deposit amount in satoshi (8-byte LE). For example:
-        // 0.0001 BTC = 10000 satoshi = 0x1027000000000000
-        bytes8 amount;
+        // Deposit amount in satoshi.
+        uint64 amount;
         // UNIX timestamp the deposit was revealed at.
         uint32 revealedAt;
         // Address of the tBTC vault.
         address vault;
+        // UNIX timestamp the deposit was swept at. Note this is not the
+        // time when the deposit was swept on Bitcoin chain but actually
+        // the time when the sweep proof was delivered on Ethereum chain.
+        uint32 sweptAt;
```

Comment:
Not sure if it's worth it, but we could use a bool here. Unless we will be using the `timestamp` for something later on.

Reply:
Not sure at this point. I'd revisit that later.